content/post/perfect-analytics-org-structure.md

```diff
@@ -0,0 +1,54 @@
+---
+author: Felix Puetsch
+title: "The perfect Analytics Org Structure"
+publishDate: "2018-08-19"
+draft: true
+tags: 
+ - analytics
+ - org structure
+ - org chart
+ - governance
+ - org
+ - teams
+---
+
+
+Have we left the Oil Age and entered the Data Age? Some would argue so. Undoubtedly, there is more data available than ever, and we are just uncovering the tip on what's possible with it. Most companies have embraced this and are employing often multiple people in an Analytics function.
```

Comment:
Style: I don't like rhetorical questions, especially not as the lead-in. My preference is to have the first paragraph really lay-out the thesis of the post. Maybe something like

> Analytics is a relatively new discipline, and many companies are still experimenting with how best to organize the work and the team. Most companies use one of four different types of organizations, each with their own drawbacks and advantages.

Then you can re-work what is currently the first paragraph a bit and make that paragraph 2.